php/libraries/NDB_BVL_Feedback.class.inc

```diff
@@ -847,7 +844,7 @@ class NDB_BVL_Feedback
         // DELETE the thread's entries
         $success = $db->delete(
             'feedback_bvl_entry',
-            array("FeedbackID" => '$feedbackID')
+            array("FeedbackID" => $feedbackID)
```

Comment:
This code was just wrong and would never delete anything. (It was caught because PHPMD said the $feedbackID variable was unused, when in reality it was just quoted as a string literal.)